### DISCUSSION TITLE
How do I upgrade the runner size from micro to medium?

### DISCUSSION THREAD

**Discussion by thiagovilla** [nektos/act#1283] [Q&A] [answered]
I need `curl` and the first time `act` prompted I went for the `micro` image (`node:16-buster-slim`), but now need to upgrade to `medium` (`catthehacker/act-node:latest`). Any kind soul?

**Comment by KnisterPeter** [comment, marked as answer]
You can update the `.actrc` file in your home directory (or remove the file).
Or to play around with the options define your runner image like this:
https://github.com/nektos/act#use-an-alternative-runner-image
